src/class.c

```diff
@@ -1224,9 +1228,6 @@ Dsymbol *InterfaceDeclaration::syntaxCopy(Dsymbol *s)
 void InterfaceDeclaration::semantic(Scope *sc)
 {
     //printf("InterfaceDeclaration::semantic(%s), type = %p\n", toChars(), type);
-    if (inuse)
-        return;
```

Comment:
Today, recursive semantic analysis is properly blocked by `semanticRun` and `doAncestorsSemantic`. Therefore, I reuse `ClassDeclaration::inuse` to detect recursive `ClassDeclaration::search` call.